core/src/main/java/org/lflang/federated/launcher/FedLauncherGenerator.java

```diff
@@ -124,7 +124,7 @@ public void doGenerate(List<FederateInstance> federates, RtiConfig rtiConfig) {
       target = user + "@" + host;
     }
 
-    shCode.append("#### Host is ").append(host);
+    shCode.append("#### Host is ").append(host).append("\n");
```

Comment:
" " followed by newline looks strange and was probably not the intent?

Reply:
??? It's `host` followed by newline.  Previously the code generator was producing:
```
#### Host is fooecho something or other
```

Reply:
You're right; I missed the first append.